res/metrics-pipeline-config.json

```diff
@@ -338,46 +299,6 @@
               ],
               "remap": null,
               "buckets": null
-            },
-            {
-              "name": "workload_dns_latency_seconds",
-              "type": "histogram",
-              "filters": [
-                {
-                  "key": "DnsId",
-                  "value": "",
-                  "type": "presence"
-                }
-              ],
-              "valueKey": "DnsLatencyMs",
-              "labels": [
-                "SrcK8S_Namespace",
-                "DstK8S_Namespace",
-                "K8S_FlowLayer",
-                "SrcSubnetLabel",
-                "DstSubnetLabel",
-                "SrcK8S_OwnerName",
-                "DstK8S_OwnerName",
-                "SrcK8S_OwnerType",
-                "DstK8S_OwnerType",
-                "SrcK8S_Type",
-                "DstK8S_Type",
-                "DnsFlagsResponseCode"
-              ],
-              "remap": null,
-              "buckets": [
-                0.005,
-                0.01,
-                0.02,
-                0.03,
-                0.04,
-                0.05,
-                0.075,
-                0.1,
-                0.25,
-                1
```

Comment:
@jpinsonneau would reducing labels and buckets help?

Reply:
It will help a bit but you will still face the same issue after some time :smile_cat: 
The safest way is to remove these

Reply:
okay, sounds good. Once you rebase, I'll give it a go.